src/config/newGame.ts

```diff
@@ -18,11 +18,21 @@ export const newGameState = () => {
             },
             research: {},
         },
-
+        playerLevelState: {
+            1: {
+                hasWatchedIntro: false,
+                hasCompletedLevel: false,
+                monthsToCompleteLevel: null,
```

Comment:
I figure it may be cool to keep track of how many months it takes to complete a level to eventually show a leaderboard or something.